README.md

```diff
@@ -655,21 +746,55 @@
     });
     ```
 
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
+
+    ```javascript
+    // bad
+    if(isJedi) {
+      fight ();
+    }
+
+    // good
+    if (isJedi) {
+      fight();
+    }
+
+    // bad
+    function fight () {
+      console.log ('Swooosh!');
+    }
+
+    // good
+    function fight() {
+      console.log('Swooosh!');
+    }
+    ```
+
   - Place 1 space before anonymous function parenthesis.
 
     ```javascript
-	// bad
-	function() {
-		var name;
-	}
+    // bad
+    function () {
```

Comment:
oops, looks like we got an errant space in the "bad" example. Should probably be `function() {` for the "bad" example.